token-client/README.md

```diff
@@ -212,6 +188,37 @@ For X.509 based authentication method using an externally managed certificate, `
   ```
 :exclamation: **DO NOT** disclose your key or secret in publicly available places e.g. repository in github.com
 
+## OAuth2ServiceConfiguration
```

Comment:
I like that we have now a separate section that explains the service config, but somehow I'm not sure about the topic sequence or the level it's kinda belongs to the each app flavor configuration sections(i.e Configuration for Spring Boot Applications) , maybe it would help when it was linked in the different app config section when talking about OAuth2ServiceConfiguration or maybe add it as first section before all other configurations so that reader has introduced himself with OAuth2ServiceConfig first and then sees the usage of it in different app types?

Reply:
I've added a link in the respective application type specific sections.